CONTRIBUTING.md

```diff
@@ -0,0 +1,388 @@
+Contributing to Gridcoin Research
+============================
+
+The Gridcoin Research project operates an open contributor model where anyone is
+welcome to contribute towards development in the form of peer review, testing
+and patches. This document explains the practical process and guidelines for
+contributing.
+
+Firstly in terms of structure, there is no particular concept of "Core
+developers" in the sense of privileged people. Open source often naturally
```

Comment:
In Gridcoin, there are core developers. I don't think we want to address "privileged"...

Reply:
Fixed.